.github/workflows/test.yaml

```diff
@@ -0,0 +1,36 @@
+name: mdet-lsst-sims
+
+on:
+    push:
+        branches: [ master ]
+    pull_request:
+        branches: [ master ]
+
+jobs:
+    build:
+        runs-on: 'ubuntu-latest'
+        container: beckermr/stackvana
```

Comment:
This stack is ancient. Do you want me to setup this container to build regularly? Otherwise i'd use the conda-forge package.

Reply:
oh, I didn't realize.  Yes please do